docs/pages/eas-update/deployment-patterns.mdx

```diff
@@ -18,25 +18,25 @@ There are three parts to consider when designing a EAS Update deployment process
 2. **Testing changes**
    - (a) We can test changes with TestFlight and Play Store Internal Track.
    - (b) We can test changes with an internal distribution build.
-   - (c) We can test changes with Expo Go or a development app.
+   - (c) We can test changes with Expo Go or a [Development Build](https://docs.expo.dev/develop/development-builds/introduction/).
```

Comment:
Just a small suggestion: shouldn't use complete URLs otherwise it forces full re-render on navigation.